src/test/resources/view/elasticview-wrong-2.json

```diff
@@ -1,9 +1,9 @@
 {
   "@id": "http://example.com/id",
   "@type": [
-    "View",
-    "ElasticView",
-    "Alpha"
+    "nxv:View",
```

Comment:
This is not needed, since where is used the [resolver context is applied](https://github.com/BlueBrain/nexus-kg/blob/master/src/test/scala/ch/epfl/bluebrain/nexus/kg/resources/ResourcesSpec.scala#L79) and that context has the neccessary alias.